rope/refactor/move.py

```diff
@@ -706,7 +706,9 @@ def rename_in_module(self, new_name, pymodule=None, imports=False, resource=None
 
     def occurs_in_module(self, pymodule=None, resource=None, imports=True):
         finder = self._create_finder(imports)
-        for occurrence in finder.find_occurrences(pymodule=pymodule, resource=resource):
+        for _occurrence in finder.find_occurrences(
+            pymodule=pymodule, resource=resource
+        ):
             return True
         return False
```

Comment:
Feels like this code could have been simplified, maybe something like:

```suggestion
        occurrences = finder.find_occurrences(pymodule=pymodule, resource=resource)
        return any(True for _ in occurrences)
```

or

```suggestion
        occurrences = finder.find_occurrences(pymodule=pymodule, resource=resource)
        sentinel = object()
        return next(occurrences, sentinel) is sentinel
```

Reply:
Rev ad9c23ed contains your second suggestion. In the future I'll use the "Commit suggestion" button.